Bio/AlignIO/StockholmIO.py

```diff
@@ -381,7 +379,7 @@ def __next__(self):
         # if present it agrees with our parsing.
 
         seqs = {}
-        ids = OrderedDict()  # Really only need an OrderedSet, but python lacks this
```

Comment:
Are sets now always ordered? At least maintain a comment that a dict is more than we need.

Reply:
I agree, it might be valuable to keep the comment in case an OrderedSet builtin comes along. Builtin set is not ordered and I haven't read that it is on the horizon either. There are packages that implement them, but I think adding dependencies for that is not desirable.

Reply:
Agreed.